src/rpc/rpc.js

```diff
@@ -55,9 +55,7 @@ module.exports = class Rpc {
   * @returns {void}
   */
   handle (message) {
-        // This guard is for backwards compatability. Having multiple responses or recieving an ack
-        // prior to a response should not actually occur
-    if (this.isComplete === true) {
+    if (this._isComplete) {
```

Comment:
this should never be called, should log it if it gets called or remove it otherwise?